src/app/pages/common/entity/entity-form/components/form-colorpicker/form-colorpicker.component.html

```diff
@@ -11,7 +11,7 @@
   <mat-form-field class="full-width">
     <input
       matInput
-      [placeholder]="config.placeholder"
+      [placeholder]="config.placeholder | translate"
```

Comment:
Changes like this are somewhat controversial. On one hand, they probably do translate more strings in the system now, on another developers should already be passing translated strings inside the config.
Let's hope everyone understands it.

Reply:
agreed to keep it until parent components are not updated to pass translated strings